File: src/main/java/io/github/chaosdave34/kitpvp/commands/DiscordCommand.java
package io.github.chaosdave34.kitpvp.commands;

import io.github.chaosdave34.kitpvp.KitPvp;
import net.kyori.adventure.text.Component;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.jetbrains.annotations.NotNull;

public class DiscordCommand implements CommandExecutor {
    @Override
    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
        sender.sendMessage(Component.text("Discord: " + KitPvp.INSTANCE.getConfig().get("discord")));
        return true;
    }
}

File: src/main/kotlin/io/github/chaosdave34/kitpvp/commands/DiscordCommand.kt
package io.github.chaosdave34.kitpvp.commands

import io.github.chaosdave34.kitpvp.KitPvp
import net.kyori.adventure.text.Component
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

class DiscordCommand : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        val discordLink = KitPvp.INSTANCE.config["discord"]
        sender.sendMessage(Component.text("Discord: $discordLink"))
        return true
    }
}